subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/recomp/GroovyRecompilationSpecProvider.java

```diff
@@ -126,7 +132,7 @@ private void processOtherChanges(PreviousCompilation previous, RecompilationSpec
             if (relativeSourceFile.isPresent()) {
                 spec.getRelativeSourcePathsToCompile().add(relativeSourceFile.get());
             } else {
-                spec.setFullRebuildCause("Can't find source file of class " + className, null);
+                spec.setFullRebuildCause("unable to find source file of class " + className, null);
```

Comment:
Can you make sure the rebuild clauses start with an uppercase letter?

Reply:
Ah - I only now saw your comment. Then let's leave them lower case.

Reply:
No we can't, the full sentence looks like this:

```
Full recompilation is required because unable to find source file of class X
```

not 

```
Full recompilation is required because Unable to find source file of class X
```